convert2rhel/subscription.py

```diff
@@ -127,12 +127,12 @@ def install_subscription_manager():
     loggerinst = logging.getLogger(__name__)
     sm_dir = os.path.join(utils.data_dir, "subscription-manager")
     if not os.path.isdir(sm_dir) or not os.listdir(sm_dir):
-        loggerinst.critical("The {0} directory does not exist or is empty."
-                            " Consider using the --disable-submgr option."
-                            " Before using it it's necessary to define a RHEL"
-                            " repo in /etc/yum.repos.d/ and pass its repoid to"
-                            " the convert2rhel through the --enable-repo"
-                            " option.".format(sm_dir))
+        loggerinst.critical("The %s directory does not exist or is empty."
+                            " Using the subsription-manager is not documented"
+                            " yet. Please use the --disable-submgr option."
```

Comment:
Typo in  --disable-submit
please, update

Reply:
This option is actually correct - no typo: https://github.com/oamg/convert2rhel/blob/master/convert2rhel/toolopts.py#L135

Reply:
oh, sorry =)